api/src/main/java/javax/enterprise/event/Event.java

```diff
@@ -123,14 +126,16 @@
 
     /**
      * <p>
-     * Fires an event asynchronously with the specified qualifiers and notifies synchronous and asynchronous observers.
+     * Fires an event asynchronously with the specified qualifiers and notifies asynchronous observers.
      * A custom {@link Executor} will be used to make asynchronous calls 
      * </p>
      *
      * @param event the event object
      * @param executor a custom executor to execute asynchronous event
      * @return a {@link CompletionStage} allowing further pipeline composition on the asynchronous operation.
-     *         if any of the synchronous or asynchronous observers notfied by this event throws an exception
+     *         Default asyncrhonous execution facility of this stage will be based on a container default {@link Executor}
+     *         specific to implementation.
+     *         if any of the synchronous or asynchronous observers notified by this event throws an exception
```

Comment:
Typo in "Default asynchronous execution" and mentioning of synchronous observers should be removed.